src/hott/05-sigma.rzk.md

```diff
@@ -223,8 +233,12 @@ This is a literate `rzk` file:
             ( first s) (first t)
             ( first (second s)) (first (second t)) p q (second (second s)) =
             ( second (second t))))
+```
+
+**Warning:** The following is the lazy definition with bad computational
+properties.
```

Comment:
We can use [admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/):

```suggestion
!!! warning

    The following definition of `triple-eq`
    is the lazy definition with bad computational properties.
```

Reply:
Oh, that's cool! Thanks for the suggestion :)